_data/glossary/sig.yaml

```diff
@@ -0,0 +1,10 @@
+id: sig
+name: SIG (Special Interest Group)
+tags:
+- community
+short-description: |
+  Manages an ongoing subproject or aspect of the larger Kubernetes open source project.
```

Comment:
Members who collectively manage an ongoing piece or aspect of the larger Kubernetes open source project.